packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -101,7 +102,25 @@ handler.getDatastore = async function getDatastore(installationId?: number) {
   if (installationId) {
     query = query.filter('installationId', installationId);
   }
-  const [prs] = await datastore.runQuery(query);
+  let prs;
+  try {
+    [prs] = await datastore.runQuery(query);
+  } catch (err) {
+    if (
+      (err as Error)
+        .toString()
+        .includes(
+          'Total timeout of API google.datastore.v1.Datastore exceeded 60000 milliseconds'
```

Comment:
Checking error types based on the message is error prone. Can we look at the request status/code first before looking at the message?

Reply:
I haven't seen any recent instances, so I wasn't able to look into the error.